transport/src/main/java/io/scalecube/transport/TransportImpl.java

```diff
@@ -369,8 +369,8 @@ private TcpClient newTcpClient(Address address) {
   private final class TransportChannelInitializer
       implements BiConsumer<ConnectionObserver, Channel> {
 
-    private static final int MAX_FRAME_LENGTH = 8192;
-    private static final int LENGTH_FIELD_LENGTH = 2;
+    private static final int MAX_FRAME_LENGTH = 2 * 1024 * 1024; // 2MB
```

Comment:
Let's create a property in `io.scalecube.transport.TransportConfig`, name it, `maxFrameLength` , let's set it by default to 2MB. LENGTH_FIELD_LENGTH  -- let's leave as hardcoded.

Reply:
Done.